### PR TITLE
Fix OS X build and execution:

### PR DESCRIPTION
**Pull request by eile**
* Remove GLU dependency
* Clean up (Pixel)Viewport code
* Use GL2 functions for shaders, not ARB
* Reduce default cache size for 32 bit programs

64 bit build still does not work on OS X. X11 OpenGL does not generate
texture IDs.

### CODE REVIEW

File: livre/Eq/Channel.cpp [hunk 1]
```diff
@@ -58,6 +52,11 @@
 #include <eq/fabric/viewport.h>
 #include <eq/fabric/pixelViewport.h>
 #include <eq/fabric/range.h>
+#include <eq/channel.h>
+#include <eq/compositor.h>
+#include <eq/frameData.h>
+#include <eq/gl.h>
+#include <eq/image.h>
```
Comment (tribal-tec) [inline livre/Eq/Channel.cpp]: eq/eq.h?

File: livre/Lib/Configuration/VolumeRendererParameters.cpp [hunk 2]
```diff
@@ -33,11 +33,18 @@ const std::string MAXLOD_PARAM = "max-lod";
 VolumeRendererParameters::VolumeRendererParameters()
     : Parameters( "Volume Renderer Parameters" )
     , renderStrategy( RS_ANY_FRAME )
-    , synchronousModeEnabled( false )
+    , synchronousMode( false )
+#ifdef __i386__
```
Comment (tribal-tec) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: really needed?

File: livre/Lib/Configuration/VolumeRendererParameters.cpp [hunk 3]
```diff
@@ -57,7 +64,7 @@ VolumeRendererParameters::VolumeRendererParameters()
     configuration_.addDescription( configGroupName_, SCREENSPACEERROR_PARAM,
                                    "Screen space error", screenSpaceError );
     configuration_.addDescription( configGroupName_, SYNCHRONOUSMODE_PARAM,
-                                   "Enable synchronous mode", synchronousModeEnabled );
+                                   "Enable synchronous mode", synchronousMode );
```
Comment (tribal-tec) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: https://github.com/BlueBrain/Livre/pull/77 goes a bit further. Remove this.

**Comment by tribal-tec** [comment]
Changelog, version bump (if merged before https://github.com/BlueBrain/Livre/pull/77)

File: livre/Lib/Configuration/VolumeRendererParameters.cpp (hunk 2 above)
Comment (eile) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: Yes, until we have a native 64 bit window system on OS X.

File: livre/Lib/Configuration/VolumeRendererParameters.cpp (hunk 3 above)
Comment (eile) [inline livre/Lib/Configuration/VolumeRendererParameters.cpp]: will rebase with #74

**Comment by eile** [comment]
Changelog?

**Comment by tribal-tec** [comment]
Changelog!

**Comment by eile** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

**Comment by bilgili** [comment]
retest this please

File: livre/Eq/Channel.cpp [hunk 4]
```diff
@@ -311,7 +302,7 @@ class Channel
 
         SelectVisibles visitor( dashTree,
                                 _currentFrustum,
-                                pixelViewport.getHeight( ),
+                                pixelViewport[3],
```
Comment (bilgili) [inline livre/Eq/Channel.cpp]: pixelViewport[3] is not very readable ?

Comment (eile) [inline livre/Eq/Channel.cpp]: pixelViewport.w() would be semantically incorrect. Type is a vmml::Vector4i.

**Pull request merged**